rskj-core/src/main/java/co/rsk/peg/Bridge.java

```diff
@@ -1033,6 +1038,31 @@ public long getFeePerKb(Object[] args)
         return bridgeSupport.getFeePerKb().getValue();
     }
 
+    public long getLockingCap(Object[] args) {
+        logger.trace("getLockingCap");
+
+        Coin lockingCap = bridgeSupport.getLockingCap();
+
+        // Parse value to BTC
+        return lockingCap.divide(Coin.COIN.getValue()).getValue();
+    }
+
+    public boolean increaseLockingCap(Object[] args) {
+        logger.trace("increaseLockingCap");
+
+        Coin newLockingCapInBTC = BridgeUtils.getCoinFromBigInteger((BigInteger) args[0]);
+        if (newLockingCapInBTC.getValue() <= 0) {
+            throw new BridgeIllegalArgumentException("Locking cap must be bigger than zero");
+        }
+        if (newLockingCapInBTC.compareTo(Coin.valueOf(21_000_000L)) > 0 ) {
```

Comment:
again, mixining satoshis/BTC is not good

Reply:
changed it to use satoshis as it was originally intended